contracts/proxies/SafeProxy.sol

```diff
@@ -31,7 +31,7 @@ contract SafeProxy {
 
     /// @dev Fallback function forwards all transactions and returns all received return data.
     fallback() external payable {
-        // solhint-disable-next-line no-inline-assembly
+        /* solhint-disable no-inline-assembly */
```

Comment:
I would revert these changes (since they are no-ops). It could cause the proxy + factory to fall on a different address for no real reason.

Reply:
Done 👍🏾